packages/kit/src/routes/config/getStateFromPath.ext.ts

```
@@ -21,28 +23,32 @@
 import type {
   InitialState,
   NavigationState,
+  ParamListBase,
   PartialState,
 } from '@react-navigation/routers';
 import escape from 'escape-string-regexp';
```

Biome (1.9.4)

[error] Do not shadow the global "escape" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)